osd/cluster_alerting_with_no_sre_access.json

```diff
@@ -0,0 +1,8 @@
+{
+    "severity": "Error",
+    "service_name": "SREManualAction",
+    "cluster_uuid": "${CLUSTER_UUID}",
+    "summary": "Attention requested: Cluster account access unavailable",
+    "description": "SRE has been notified of problems on your cluster, but cannot investigate because cloud provider access has been removed. If you intended to terminate this cluster then please delete the cluster in the Red Hat console. Otherwise file a support request.",
```

Comment:
I would add a small comment saying that the cluster would be in limited support and silenced while we regain access